src/api/app.py

```diff
@@ -205,6 +207,24 @@ def log_out():
 def add_user_event():
     return event_controller.add_event(json.loads(request.data))
 
+@app.route('/api/addcourse', methods=['POST'])
+def add_student_course():
+    info = request.get_json()
+    resp, error = course_select.add_selection(info['name'], info['semester'], info['uid'], info['cid'])
+    return Response(status=200) if not error else Response(error, status=500)
+
+
+@app.route('/api/removecourse', methods=['POST'])
+def remove_student_course():
+    info = request.get_json()
+    resp, error = course_select.remove_selection(info['name'], info['semester'], info['uid'], info['cid'])
+    return Response(status=200) if not error else Response(error, status=500)
+
+@app.route('/api/getcourses', methods=['POST'])
```

Comment:
reason for not doing methods=['GET']?

Reply:
Not easy to pass an argument to 'GET', if there is a way I wasn't using, lmk

Reply:
if all you're passing to the GET is 
```
const info = {'uid': this.userID};
```
then you can just include that as 
```
uid => client.get('/getcourses', {
    params: {
        "uid": uid
    }
})
```
and update your flask handler to use `request.args['uid']` instead

Reply:
Done